cmp/grid/Grid.js

```diff
@@ -144,9 +140,11 @@ export class Grid extends Component {
                 className: this.getClassName(
                     'ag-grid-holder',
                     XH.darkTheme ? 'ag-theme-balham-dark' : 'ag-theme-balham',
-                    compact ? 'xh-grid-compact' : 'xh-grid-standard',
-                    treeMode && this._isHierarchical ? 'xh-grid-hierarchical' : '',
-                    !isMobile && showHover ? 'xh-grid-show-hover' : ''
+                    treeMode && this._isHierarchical ? 'xh-grid--hierarchical' : 'xh-grid--flat',
+                    compact ? 'xh-grid--compact' : 'xh-grid--standard',
+                    rowBorders ? '' : 'xh-grid--no-row-borders',
+                    stripeRows ? '' : 'xh-grid--no-stripes',
```

Comment:
We could go ahead here and flip the sense of these classes to consistently add formatting, especially since it looks like we'll land on a default of no borders.